packages/solid/.storybook/preview-head.html

```diff
@@ -42,4 +42,8 @@
   .sb-main-padded {
     overflow: hidden;
   }
+
+  .sb-show-main{
+    background-color: #1b1c1d;
+  }
```

Comment:
what's this doing?

Reply:
Thats for the popout - sets the background so it's not white.

https://rdkcentral.github.io/solid-ui/iframe.html?args=&id=components-artwork--image&viewMode=story